README.md

```diff
@@ -35,7 +35,7 @@ we created this implementation of Language Server.
 
 ### Build extension from source
 
-* Run `npx vsce package` in the root directory.
+* Run `npx @vscode/vsce package` in the root directory.
```

Comment:
`vsce` package is now deprecated and should be replaced with `@vscode/vsce`. See https://www.npmjs.com/package/vsce